cpp/src/glm/qn/simple_mat/sparse.hpp

```diff
@@ -94,7 +96,7 @@ struct SimpleSparseMat : SimpleMat<T> {
     // to swap arguments A and B in cusparseSpMM.
     cusparseDnMatDescr_t descrC;
     auto order = C.ord == COL_MAJOR ? CUSPARSE_ORDER_ROW : CUSPARSE_ORDER_COL;
-    RAFT_CUSPARSE_TRY(raft::sparse::cusparsecreatednmat(
+    RAFT_CUSPARSE_TRY(raft::sparse::detail::cusparsecreatednmat(
```

Comment:
Just curious if there is an issue to track removal of usage of `detail` APIs in the code base?